config/sync/views.view.events.yml

```diff
@@ -0,0 +1,229 @@
+uuid: 64c2acda-1c36-4843-a6ae-35b06b00c172
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_event_date
+    - search_api.index.events
+  module:
+    - datetime_range
+    - search_api
+id: events
+label: Events
+module: views
+description: ''
```

Comment:
Consider if you can provide a description that describes the purpose of the view.